### ISSUE TITLE
more of a question than an issue ...

### ISSUE THREAD

**Issue by ecoquant** [kawesomekhan/covid-indoor#74]
I am rewriting the Python code in **R** so I can do sensitivity analysis on some of the parameters.

I'm pretty sure the answer is "yes" but I just wanted to check that the Python in the Github is in every way consistent with the spreadsheet at http://www.mit.edu/~bazant/COVID-19/ and the online app linked from the same. 

Thank you. Great tools.

**Comment by ecoquant** [comment]
I have completed a basic implementation of the model in **R**. This was done by beginning with the Python sketch, but finishing the calculation details by consulting the spreadsheet supplied in association with the PNAS article and http://www.mit.edu/~bazant/COVID-19/. The spreadsheet was used to develop truth data for validation, as well as checking intermediate results. I have placed these at [this location](https://drive.google.com/drive/folders/1teYwKoDkFZXC474ZcYquP19YwQXuUKxp?usp=sharing),

The version of the **R** code there has been validated against the spreadsheet, but it does not, as yet, contain any sensitivity analysis or plots.  I will do that beginning tomorrow.  The intent is to vary key parameters, such as risk tolerance, or mask effectiveness, or mask fit and see how the maximum number of people varies accordingly.  In fact what will be done is a large number of states, an ensemble, will be generated, and then this set of states can be interrogated for various questions regarding the joint distribution. 

I will update the code at the given location.
